components/brave_extension/extension/brave_extension/background/api/shieldsAPI.ts

```diff
@@ -164,23 +164,32 @@ export const setAllowScriptOriginsOnce = (origins: Array<string>, tabId: number)
 
 export type GetViewPreferencesData = {
   showAdvancedView: boolean
+  statsBadgeVisible: boolean
 }
 
 const settingsKeys = {
-  showAdvancedView: { key: 'brave.shields.advanced_view_enabled', type: chrome.settingsPrivate.PrefType.BOOLEAN }
+  showAdvancedView: { key: 'brave.shields.advanced_view_enabled', type: chrome.settingsPrivate.PrefType.BOOLEAN },
+  statsBadgeVisible: { key: 'brave.shields.stats_badge_visible', type: chrome.settingsPrivate.PrefType.BOOLEAN },
```

Comment:
lint: comma (even though I personally like this style :-( )

Reply:
thought our linter would spot that. updated